copilot/post-award-celery/manifest.yml

```diff
@@ -40,6 +40,7 @@ variables:
     from_cfn: ${COPILOT_APPLICATION_NAME}-${COPILOT_ENVIRONMENT_NAME}-postawardsuccessfulfilesBucketName
   AWS_S3_BUCKET_FAILED_FILES:
     from_cfn: ${COPILOT_APPLICATION_NAME}-${COPILOT_ENVIRONMENT_NAME}-postawardfailedfilesBucketName
+  ACCOUNT_STORE_API_HOST: "http://fsd-account-store:8080"
```

Comment:
Don't think this is needed here as Celery instance shouldn't be running this script?

Reply:
I removed it from Celery manifest.
I thought we want to keep these consistent. But it's likely not going to be needed on Celery.

Reply:
My preference would be for all of the config to be consistently applied between both apps 🤷‍♂️  They are the same codebase - it feels likely to just trip people up in the future if config is different between them (or some values are just unset).